include/swift/AST/Identifier.h

```diff
@@ -592,6 +592,21 @@ class DeclName {
                             "only for use within the debugger");
 };
 
+/// Apply a macro FAMILY(Name, Prefix) to all ObjC selector families.
+#define FOREACH_OBJC_SELECTOR_FAMILY(FAMILY)       \
```

Comment:
Now that this is going into a .h file (instead of a .cpp file)... what do you think about putting it into a .def. It would be more consistent.

Reply:
That's the "proper" way to structure it, but it seems like overkill for a five-item list that will never change and that's used in only two places. Do you think it's worth doing anyway?

Reply:
Yes. Consistency is important in a large code base.

Reply:
The larger fear is that someone will see this and cargo-cult it for other cases where it is inappropriate.